### PR TITLE
Add Ethereum Name Service (ENS)

### PR DESCRIPTION
**Pull request by ligi**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no direct affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection

---------------------------------------------------------------------

### Editing an existing topic or collection

I'm suggesting these edits to an existing topic or collection:
- [x] Image (and my file is `*.png`, square, dimensions 288x288)
- [x] Content (and my changes are in `index.md`)
> Please replace this line with an explanation of why you think these changes should be made.

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Please replace this line with an explanation of why you think this topic or collection should be curated.

---------------------------------------------------------------------

### CODE REVIEW

File: topics/ens/index.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+display_name: Ethereum Name Service (ENS)
+logo: ens.png
+released: May 4, 2017
+short_description: ENS is a name service build on Ethereum.
+topic: ens
+related: ethereum, blockchain, cryptocurrency
+url: https://www.ens.domains
```
Comment (Arachnid) [inline topics/ens/index.md]: No www here!

Comment (ligi) [reply topics/ens/index.md]: thanks - removed

**Comment by filips123** [comment]
Can you add both `ens` and `ethereum-name-service` tags? Both of them are quite used in ENS-related projects.

**Comment by ligi** [comment]
where?

**Comment by filips123** [comment]
Do you mean where to add them or where are they used?

You can add them as `aliases`. See [here](https://github.com/github/explore/blob/master/topics/bitcoin/index.md) how `bitcoin` tag does this.

Also `ens` tag has [56 repositories](https://github.com/topics/ens) and `ethereum-name-service` has [5 repositories](https://github.com/topics/ethereum-name-service). Even if `ens` is more popular, both are existing and should be handled properly.

**Comment by kenyonj** [comment]
💯 🍨 Thank you for your contribution! I'll merge this in as soon as CI is 🍏

**Pull request merged**